Jenkinsfile

```diff
@@ -0,0 +1,36 @@
+@Library('podTemplateLib')
+import net.santiment.utils.podTemplates
+
+properties([buildDiscarder(logRotator(artifactDaysToKeepStr: '30', artifactNumToKeepStr: '', daysToKeepStr: '30', numToKeepStr: ''))])
+
+slaveTemplates = new podTemplates()
+
+slaveTemplates.dockerTemplate { label ->
+  node(label) {
+    stage('Build') {
+      container('docker') {
+        def scmVars = checkout scm
+
+        if (scmVars.GIT_TAG_NAME.contains('relay')) {
+          withCredentials([
+            string(
+              credentialsId: 'SECRET_KEY_BASE',
+              variable: 'SECRET_KEY_BASE'
+            ),
+            string(
+              credentialsId: 'aws_account_id',
+              variable: 'aws_account_id'
+            )
+          ]) {
+            // We are building two docker images - for stage and prod respectively. The app requires an env var BACKEND_URL set at build time.
+            def awsRegistry = "${env.aws_account_id}.dkr.ecr.eu-central-1.amazonaws.com"
+            docker.withRegistry("https://${awsRegistry}", "ecr:eu-central-1:ecr-credentials") {
+              sh "docker build -t ${awsRegistry}/san-rewards-relay:${scmVars.GIT_COMMIT} ."
+              sh "docker push ${awsRegistry}/san-rewards-relay:${scmVars.GIT_COMMIT}"
```

Comment:
and push to master as well `sh "docker push ${awsRegistry}/san-rewards-relay:${env.BRANCH_NAME}"`